### PR TITLE
Springer header js

### PR DESCRIPTION
**Pull request by davidpauljunior**

### CODE REVIEW

File: toolkits/springer/packages/springer-header/js/enhanced-menu.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import {makeArray} from '@springernature/global-context/js/helpers';
+import {createDropdown} from '@springernature/springer-dropdown/js/create-dropdown';
+
+const createMenuDropdown = menu => {
+	menu.classList.add('u-hide-at-lt-lg');
+
+	const items = menu.querySelectorAll('li');
+	const links = makeArray(items).map(item => item.querySelector('a'));
+
+	const dropdownEl = createDropdown({
```
Comment (ricky-lau) [inline toolkits/springer/packages/springer-header/js/enhanced-menu.js]: just got a linting error about `dropdownEl` should be named `dropdownElement`

Comment (ricky-lau) [reply toolkits/springer/packages/springer-header/js/enhanced-menu.js]: ah you can see them in the travis build now..

Comment (davidpauljunior) [reply toolkits/springer/packages/springer-header/js/enhanced-menu.js]: 😭

**Comment by ricky-lau** [comment]
Does the readme need updating with `enhanced-menu`?

**Comment by davidpauljunior** [comment]
This has a dependency on another component which shouldn't happen.
This is now shared code within the product.

**Pull request closed without merging**